src/components/BaseComponents/Link/Link.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function Link(props) {
+  const { onClick, children, attributes } = props;
+
+  return (
+    <div className='govuk-button-group'>
```

Comment:
add this div with className='govuk-button-group' to the parent component. Link component should not have this

Reply:
Use button component with varient=link instead of creating a new component for link

Reply:
have used button component for both backlink and link